src/main/scala/com/cloudera/sparkts/TimeSeriesRDD.scala

```diff
@@ -268,7 +269,7 @@ class TimeSeriesRDD[K](val index: DateTimeIndex, parent: RDD[(K, Vector)])
             i += 1
           }
           dtLoc += 1
-          ((dtLoc - 1, partitionId * maxChunkSize + chunkId), new DenseVector(arr))
+          ((dtLoc - 1, partitionId * maxPartitionSize + chunkId), new DenseVector(arr))
```

Comment:
A disadvantage of this approach is that it triggers computation of the RDD here.  Can we instead just make the tuple a triple where partitionId and chunkId are two separate components?